test_scripts/Polices/user_consent_of_Policies/100_ATF_TC_P_isAppPermissionsRevoked_true.lua

```diff
@@ -148,15 +148,13 @@ function Test:Precondition_PTU_revoke_app_group()
             end)
         end)
     end)
-
-  EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "NONE", systemContext = "MAIN", audioStreamingState = "NOT_AUDIBLE" })
 end
 
 --[[ Test ]]
 commonFunctions:newTestCasesGroup("Test")
 function Test:TestStep_Activate_app_isAppPermissionRevoked_true()
   local RequestIdActivateAppAgain = self.hmiConnection:SendRequest("SDL.ActivateApp", { appID = HMIAppID })
-  EXPECT_HMIRESPONSE(RequestIdActivateAppAgain, { result = { code = 0, method = "SDL.ActivateApp", isAppRevoked = true, isAppPermissionsRevoked = true}})
+  EXPECT_HMIRESPONSE(RequestIdActivateAppAgain, { result = { code = 0, method = "SDL.ActivateApp", isAppRevoked = false, isAppPermissionsRevoked = true, appRevokedPermissions = { {name = "DrivingCharacteristics"} }}})
```

Comment:
@VVeremjova  In defect "[GENIVI] SDL doesn't send appRevokedPermissions in SDL.ActivateApp" we have clarification in comments, please check.